_vector-search/getting-started/vector-search-options.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Bringing or generating embeddings
+parent: Getting started
+nav_order: 20
+quickstart_cards:
+  - heading: "Getting started with vector search"
+    description: "Use raw vectors or embeddings generated outside of OpenSearch"
+    link: "/vector-search/getting-started/pre-generated-embeddings/"
+tutorial_cards:
+  - heading: "Generating embeddings within OpenSearch"
+    description: "Automatically convert data to embeddings within OpenSearch"
+    link: "/vector-search/getting-started/auto-generated-embeddings/"
+  - heading: "Getting started with semantic and hybrid search"
+    description: "Learn how to implement semantic and hybrid search"
+    link: "/vector-search/getting-started/tutorials/neural-search-tutorial/"
+pre_items:
+  - heading: "Generate embeddings"
+    description: "Generate embeddings outside of OpenSearch using your favorite embedding utility."
+  - heading: "Create an OpenSearch index"
+    description: "Create an OpenSearch index to upload your embeddings."
+    link: "/vector-search/creating-vector-index/#pre-generated-embeddings-or-raw-vectors"
+  - heading: "Ingest embeddings"
+    description: "Ingest your embeddings into the index."
+    link: "/vector-search/ingesting-data/#raw-vector-ingestion"
+  - heading: "Search embeddings"
+    description: "Search your embeddings using vector search."
+    link: "/vector-search/searching-data/#searching-pre-generated-embeddings-or-raw-vectors"
+auto_items:
+  - heading: "Configure an embedding model"
+    description: "Configure a machine learning model that will automatically generate embeddings from your text at ingest time and query time."
+    link: "/ml-commons-plugin/integrating-ml-models/"
+  - heading: "Create an OpenSearch index"
+    description: "Create an OpenSearch index to upload your text."
+    link: "/vector-search/creating-vector-index/#converting-data-to-embeddings-during-ingestion"
+  - heading: "Ingest text"
+    description: "Ingest your text into the index."
+    link: "/vector-search/ingesting-data/#converting-data-to-embeddings-during-ingestion"
+  - heading: "Search text"
+    description: "Search your text using vector search. Query text is automatically converted to vector embeddings and compared to document embeddings."
+    link: "/vector-search/searching-data/#searching-auto-generated-embeddings"
+---
+
+# Bringing or generating embeddings
```

Comment:
Can this page be moved into Vector Search Basics? Im not sure it needs its own page.

Reply:
Having this as a separate page allows users to discover this page while scanning the left nav. Vector search basics provides info about vector search for beginners: experienced users might not look there. This page provides info about the options they have, so I think it's warranted as a separate page.